auto_update_tests.py

```diff
@@ -41,6 +41,10 @@
         print ' '.join(cmd)
         actual = run_command(cmd)
         with open(os.path.join('test', wasm), 'w') as o: o.write(actual)
+        if 'debugInfo' in asm:
+          cmd += ['--source-map', os.path.join('test', wasm + '.map'), '-o', 'a.wasm']
```

Comment:
the code above generates proper data for test;`a.wasm` is just a file we can delete -- we only need source map from this run.